frontend/front/src/pages/Public/Components/CanonicalizationLoader.js

```diff
@@ -0,0 +1,60 @@
+import React, { forwardRef, useEffect, useMemo, useState } from "react";
+import { useGetHomeQuery } from "../../../api/apiSlice";
+import Loader from "../../../components/Loader";
+import { Box } from "@mui/material";
+
+const CanonicalizationLoader = forwardRef(({ homeId, onResolved }, ref) => {
+  const [stopPolling, setStopPolling] = useState(false);
+  const {
+    data: validationData,
+    isLoading: isLoadingValidation,
+    isError: validationError,
+  } = useGetHomeQuery(homeId, { pollingInterval: stopPolling ? 0 : 300 });
```

Comment:
> pollingInterval

thank you rtk query :clap: 

Any particular reason the interval is 300ms (not that that's not a fine interval)

Reply:
Amen, pretty sweet API there!

I am hitting an average of 2-3 seconds for canonicalization, so it's really just a number that will allow the verification to feel roughly as fast as the server wants to respond without making a crazy number of requests.